lib/ansible/parsing/vault/__init__.py

```diff
@@ -240,6 +239,7 @@ class PromptVaultSecret(VaultSecret):
     default_prompt_formats = ["Vault password (%s): "]
 
     def __init__(self, _bytes=None, vault_id=None, prompt_formats=None):
+        super(PromptVaultSecret, self).__init__(_bytes=_bytes)
         self._bytes = _bytes
```

Comment:
Now that we're calling super, we should remove setting self._bytes in the PromptVaultSecret class, shouldn't we?